bin/_duo

```diff
@@ -66,6 +67,8 @@ var program = new Command('duo')
   .option('-C, --no-cache', 'enable or disable the cache during build', true)
   .option('-d, --development', 'include development dependencies.', false)
   .option('-g, --global <name>', 'expose entry as a global <name>', null)
+  .option('-i, --inline-source-map', 'include source-maps inline to the entry file', false)
```

Comment:
I preferred to not have to specify a source-map path, having it automatically drop it in the right place was more important to me. (imo)